docker/Dockerfile

```diff
@@ -17,21 +17,25 @@
 
 
 # Args used in FROM commands must come first
-ARG FROM_IMAGE="gpuci/miniforge-cuda"
-ARG CUDA_VER=11.5
+ARG FROM_IMAGE="rapidsai/mambaforge-cuda"
+ARG CUDA_SHORT_VER=11.5
+ARG CUDA_PKG_VER=11-5
+ARG CUDA_VER=11.5.1
```

Comment:
Either build up from Major.Minor.Rev or parse out the whole string into separate args.